docs/cli/migration/lambda-layers-update.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Lambda layer behavior updates
+description: Starting from Amplify CLI version 5.0.0, the Lambda layers behavior in Amplify CLI has been updated significantly
+---
+Amplify has updated the way Lambda layer versions are managed with Amplify CLI version 5.0.0. Amplify CLI enables you to configure Lambda layers to pull common code & assets for your Lambda function into a centralized location. 
```

Comment:
```suggestion
Amplify has updated the way [Lambda layer](~/cli/function/layers.md) versions are managed with Amplify CLI version 5.0.0. Amplify CLI enables you to configure Lambda layers to pull common code & assets for your Lambda functions into a centralized location. 
```

Reply:
Just to add a ref to the Layers docs since the search does not pick up those anchors in the autocomplete.